src/test/java/macrobase/ingest/CSVIngesterTest.java

```diff
@@ -1,26 +1,63 @@
 package macrobase.ingest;
 
+import java.io.IOException;
+import java.util.ArrayList;
+
 import com.google.common.collect.Lists;
-import macrobase.conf.ConfigurationException;
-import macrobase.conf.MacroBaseConf;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+import macrobase.conf.ConfigurationException;
+import macrobase.conf.MacroBaseConf;
+import macrobase.datamodel.Datum;
 
-import java.io.IOException;
-import java.util.ArrayList;
 
 public class CSVIngesterTest {
     private static final Logger log = LoggerFactory.getLogger(CSVIngesterTest.class);
 
-    public void simpleTest() throws IOException, ConfigurationException {
```

Comment:
@mamikonyana how did you run this test previously? I made it into a JUnit test to run it.